common-items.lic

```diff
@@ -44,7 +44,6 @@ module DRCI
     /You just can't/,
     /push you over the item limit/,
     /You stop as you realize the .* is not yours/,
-    /Stow what/,
```

Comment:
That's fine to remove it here; however, then that match string needs to be added to `@@stow_item_failure_patterns`, which currently is just the combination of other pattern lists:
```ruby
  @@stow_item_failure_patterns = [
    *@@get_item_failure_patterns,
    *@@put_away_item_failure_patterns,
  ]
 ```

Reply:
Then we can add the `Stow what` pattern to `@@put_away_item_failure_patterns`

It doesn't belong in the one for get failures.

Reply:
Good catch, I agree. Would you mind adding `/Stow what/,` as the first item in `@@put_away_item_failure_patterns` at line 112? Thanks

Reply:
Sure.  I'm still not sure why you are adding the get item success/failure messages in the stow item success/failure messages.

```  @@stow_item_success_patterns = [
    *@@get_item_success_patterns,
    *@@put_away_item_success_patterns
  ]

  @@stow_item_failure_patterns = [
    *@@get_item_failure_patterns,
    *@@put_away_item_failure_patterns,
  ]```

Reply:
Thanks. I put the get/put success in the stow success, and the get/put failure in the stow failure because a stow command is a simultaneous "get and put". I figured it was a nice way to avoid duplicating strings.

Reply:
In all honesty, I likely meant to put the `/^Stow what/` in the put failure patterns and accidentally added it in the get failure

Reply:
Okay I see.  I was thinking more along the lines of stowing things already in your hands.

It should be fine the way you have it then.